### ISSUE TITLE
Proposed new namespace cdm.base.staticdata.common

### ISSUE THREAD

**Issue by tomhealey-icma** [finos/common-domain-model#2118]
ISOCurrencyCodeEnum is currently located in namespace cdm.base.staticdata.asset.common but currency is not asset specific and we need to add ISO County Codes and other common ISO data. I recommend adding a new namespace, namespace cdm.base.staticdata.common for these enums.

**Comment by chrisisla** [comment]
Agreed Currency is not specific to assets so should be extracted from that namespace. I'm not that keen on a new cdm.base.staticdata.common namespace though as I think it will end up a dumping ground for anything that does not easily fit anywhere else (and encourage people to be lazy when adding enums etc. to the model).

Would it be better to have a cdm.base.staticdata.currency or .country namespace?

**Comment by eteridvalishvili** [comment]
Hi @tomhealey-icma Can this issue be closed?

**Comment by tomhealey-icma** [comment]
Not yet. When we submit a PR we'll need link it to an issue.

**Comment by eteridvalishvili** [comment]
Hi @tomhealey-icma Was the PR submitted? Please let me know if this issue is good to close. If not, maybe we can put it on the agenda for the upcoming WGs.
